src/lib/trivy/trivy-collector.spec.ts

```diff
@@ -59,6 +60,17 @@ describe('TrivyCollector', () => {
       expect(collector.parseResults).toHaveBeenCalledTimes(1);
       expect(collector.parseResults).toHaveBeenCalledWith('TEST_OUTPUT');
     });
+
+    it('should error when image cannot be found', async () => {
+      collector._argv = {
+        'image-name': 'TEST_IMAGE',
+      } as any;
+
+      theSpy.and.callThrough(); // this removes the return value in line 16
```

Comment:
Can you remove `line 16` in the comments? It will quickly become obsolete and misleading.